docs/user/setup.rst

```diff
@@ -18,6 +18,7 @@ Prerequisites
 -------------
 
 * `Docker <http://www.docker.com>`_
+* (Linux) `Docker Compose (>=1.14) <https://docs.docker.com/compose/install/>`_
```

Comment:
Can you move it to separate subsection for Linux?

Reply:
or `Docker Compose (>=1.14 **Linux only**) <https://docs.docker.com/compose/install/>`_